bot/cogs/config_verifier.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from discord.ext.commands import Cog
+
+from bot import constants
+from bot.bot import Bot
+
+
+log = logging.getLogger(__name__)
+
+
+class ConfigVerifier(Cog):
+    """Verify config on startup."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.bot.loop.create_task(self.verify_channels())
```

Comment:
While we used to do this (and probably had to us this prior to Python 3.7), I think we should switch to using the higher level asyncio API with `asyncio.create_task` instead of using the lower-level loop API of `loop.create_task`. I think this is as good a point as any to introduce a new habit.

I think it's also a good idea to keep a reference to the task around. That will make it easier to debug since we can inspect exceptions and so on if something breaks.

So, would suggest, **with an additional `import asyncio` at the top**, this:

```suggestion
        self.verification_task = asyncio.create_task(self.verify_channels())
```

If we merge this after the Python 3.8 migration, we can also add an explicit `name` kwarg to give the task a beautiful repr.

Reply:
This fails as the bot's event loop is not running yet and it can't fetch it if I'm understanding it right. Should it at least be assigned to a var?

Reply:
Right, that is true, I guess.